staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go

```diff
@@ -81,6 +82,9 @@ type APIGroupVersion struct {
 
 	// OpenAPIConfig lets the individual handlers build a subset of the OpenAPI schema before they are installed.
 	OpenAPIConfig *openapicommon.Config
+
+	// Authorizer determines whether a user is allowed to make a certain request.
+	Authorizer authorizer.Authorizer
```

Comment:
Perhaps place right above Admit? And add to the description--folks will expect authz has already been called, so under what circumstances will this be called again?